pkg/routes/logs.go

```diff
@@ -43,7 +44,12 @@ func (l Logs) Install(c *restful.Container) {
 func logFileHandler(req *restful.Request, resp *restful.Response) {
 	logdir := "/var/log"
 	actual := path.Join(logdir, req.PathParameter("logpath"))
-
+	//According to NCC-E003660-B4Y. Although http.Server explicitly sanitizes its input,
```

Comment:
can we add an unit test?

Reply:
I have added unit tests for two exceptional cases. Since the result of the function http.ServeFile depends on the environment, I did not specifically add tests for it.If there are any further issues, please kindly remind me.